components/dependency_injection/advanced.rst

```diff
@@ -63,11 +53,19 @@ Here is an example:
         $definition->setPublic(false);
         $container->setDefinition('foo', $definition);
 
-Now that the service is private, you *should not* call (should not means, this
-*might* fail, see the explaination above)::
+What makes private services special is that they are converted from services
+to inlined instantiations (e.g. ``new PrivateThing()``) if they are only
+injected once. This increases the container's performance.
```

Comment:
well I know it isn't very great english (well, at least it isn't very good dutch), I like "What makes private services special is that, if they are only injected once, they are converted from services to inlined instantiations (e.g. ``new PrivateThing()``)." more